interpreter/binary/encode.ml

```diff
@@ -224,8 +225,7 @@ let encode m =
       | Const {it = I64 c; _} -> op 0x42; vs64 c
       | Const {it = F32 c; _} -> op 0x43; f32 c
       | Const {it = F64 c; _} -> op 0x44; f64 c
-      | Const {it = V128 c; _} ->
-        failwith "TODO v128"
+      | Const {it = V128 c; _} -> simd_op 0x02; Bytes.iter (put s) (V128.to_bits c)
```

Comment:
Nit: for symmetry with the other value types, factor out encoding of the value itself into a separate `v128` function.

Btw, V128.to_bits should return a `string`, not `bytes`. OCaml's choice of type names may be misleading, but strings are byte strings, while the latter type is simply a mutable version of string, which is not what you want for this purpose.

Reply:
I see, thanks for clarifying. So the reason we want `to_bits` to return a `string` is so that it's immutable and callers cannot modify the underlying bytes?

Reply:
It looks like we can change https://github.com/WebAssembly/simd/blob/master/interpreter/exec/simd.ml#L36 to `module Make (Rep : RepType) : S with type bits = String.t =`
and have `type bits = string` below. Does that sound reasonable?

Reply:
Yes and yes. You can also use `string` instead of `String.t`.

Reply:
Done!